web/package-lock.json

```diff
@@ -10302,9 +10407,9 @@
       "dev": true
     },
     "imjoy-core": {
-      "version": "0.12.3",
-      "resolved": "https://registry.npmjs.org/imjoy-core/-/imjoy-core-0.12.3.tgz",
-      "integrity": "sha512-tLzc+7Az4hjeQlV2ypzrm1dNtQNctjvRhIWu1vAQo5//r2PlOTla1/KUdl+4AytylBtcnwEBPAA/KuugNP2mIQ==",
+      "version": "0.12.5-a",
```

Comment:
There's a dash here that's not present in `package.json`.

Reply:
Aha, that’s the reason, let me add that to package.json

Reply:
That fixes the issue!